src/rmon/rmoncmds.c

```diff
@@ -25,7 +25,7 @@ static FUNPTR dispatchTable[] = {
     NotImplemented,      NotImplemented,      NotImplemented,       NotImplemented,    __rmonGetSRegs,
     __rmonSetSRegs,      __rmonGetVRegs,      __rmonSetVRegs,       NotImplemented,
 };
-
+/*
```

Comment:
Let's not leave this as a future landmine, please either remove all of rmon or fix the error properly by changing the `FUNPTR` type in `rmonint.h` to
`typedef int (*FUNPTR)(KKHeader* request);`

Reply:
rmon removed